### ISSUE TITLE
Version 2.51 do not prevent computer to go sleep

### ISSUE THREAD

**Issue by Tansael** [develsoftware/GMinerRelease#205]
I've updated to the v2.51 now miner doesn't prevent the computer to go into sleep mode.  When I'm using the computer it's fine but when I leave it alone the computer goes to sleep mode.

**Comment by foomip** [comment]
Isn't that something you would set up in the power/performance settings of your OS?

**Comment by Tansael** [comment]
Yes, I can set up from there, too. But this doesn't happen in v2.49. I just want to mention that. This isn't a big issue.

**Comment by Mugen4** [comment]
I was using v249 and this morning found my computer sleeped for first time in several days. Then I checked for updates and installed the v251. Two hours later, my computer fell sleep again. I think it is not something about 251 version.

**Comment by Tansael** [comment]
So, it looks like a coincidence, maybe it's caused by a Windows update.
